taskt/Core/Automation/Commands/RunScriptCommand.cs

```diff
@@ -32,26 +48,51 @@ public RunScriptCommand()
 
         public override void RunCommand(object sender)
         {
-            {
-                System.Diagnostics.Process scriptProc = new System.Diagnostics.Process();
+            System.Diagnostics.Process scriptProc = new System.Diagnostics.Process();
 
-                var scriptPath = v_ScriptPath.ConvertToUserVariable(sender);
-                scriptProc.StartInfo.FileName = scriptPath;
-                scriptProc.StartInfo.WindowStyle = System.Diagnostics.ProcessWindowStyle.Hidden;
-                scriptProc.Start();
-                scriptProc.WaitForExit();
+            string scriptPath = v_ScriptPath.ConvertToUserVariable(sender);
+            string scriptType = v_ScriptType.ConvertToUserVariable(sender);
+            string scriptArgs = v_ScriptArgs.ConvertToUserVariable(sender);
```

Comment:
In the case that there's more than one argument, the ConvertToUserVariable will assume it's all just one variable and try to look it up. I don't think this would be an issue with PowerShell scripts if you can put them all in one line/variable, but will that work with python arguments too? One thing you can do is give the user the option to delimit the arguments with a semicolon, split scriptArgs, and then convert them individually. There's an example of how this is done in the OutlookEmailCommand for attachment variables.